### ISSUE TITLE
Custom TabBar: update for position not being called when selecting item

### ISSUE THREAD

**Issue by mojtaba-almoussawi** [uias/Tabman#309]
i have subclassed TabmanBar and create my subclass, however the two functions : 
` override func update(forPosition position: CGFloat,
                         direction: PageboyViewController.NavigationDirection,
                         indexRange: Range<Int>,
                         bounds: CGRect`

`open func update(forPosition position: CGFloat,
                     direction: PageboyViewController.NavigationDirection,
                     indexRange: Range<Int>,
                     bounds: CGRect)`

not called when any item in the TabBar selected, they are called only once the tabbar will be presented.

**Comment by aKhanTechmantra** [comment]
I am also facing the same issue. Is there any workaround to fix this issue?

**Comment by msaps** [comment]
Closing due to the release of [Tabman 2](https://github.com/uias/Tabman/releases/tag/2.0.0) - which should have resolved this issue. 🎉 

If not, please open a new issue. 🙂
